charts/apisix/Chart.yaml

```diff
@@ -31,20 +31,20 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.3.4
+version: 0.3.5
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 2.6.0
+appVersion: 2.7.0
 
 dependencies:
   - name: etcd
     version: 6.2.6
     repository: https://charts.bitnami.com/bitnami
     condition: etcd.enabled
   - name: apisix-dashboard
-    version: 0.1.4
+    version: 0.1.5
```

Comment:
The apisix-dashboard chart `0.1.5` hasn't released yet. This change will break the helm dependencies installation.